service/chartconfig/v1/helm/helm_test.go

```diff
@@ -10,6 +10,67 @@ import (
 	helmrelease "k8s.io/helm/pkg/proto/hapi/release"
 )
 
+func Test_DeleteRelease(t *testing.T) {
+	testCases := []struct {
+		description  string
+		namespace    string
+		releaseName  string
+		releases     []*helmrelease.Release
+		errorMatcher func(error) bool
+	}{
+		{
+			description:  "case 0: try to delete non-existent release",
+			releaseName:  "chart-operator",
+			releases:     []*helmrelease.Release{},
+			errorMatcher: IsReleaseNotFound,
+		},
+		{
+			description: "case 1: delete basic release",
+			releaseName: "chart-operator",
+			releases: []*helmrelease.Release{
+				helmclient.ReleaseMock(&helmclient.MockReleaseOptions{
+					Name:      "chart-operator",
+					Namespace: "default",
+				}),
+			},
+		},
+		{
+			description: "case 2: try to delete release with wrong name",
```

Comment:
Is that over the top? Technically its like `case 0`, just that theres another release installed.

Reply:
No I like it. It checks we're deleting the correct release and not just the first one for example.